### DISCUSSION TITLE
Reverse proxy for Apache

### DISCUSSION THREAD

**Discussion by olewolf** [slskd/slskd#890] [General]
Has anyone managed to write a reverse proxy section for Apache?

I realize that if I want slskd to be accessed via a subfolder ("http://my.domain/slskd"), then I must set the urlBase key accordingly in the slskd configuration. But, the web page remains stuck at "Lost connection to slskd / Refresh to reconnect." I imagine it has to do with websockets forwarding in Apache, and while I have had success setting up a reverse proxy for other websocket apps, I need help figuring out how to do this with slskd.

**Comment by olewolf** [comment]
Well, I'll be. Just an hour later it turned out that the reason I couldn't get it to work may have been entirely unrelated to Apache. In any event, it seems that it was as simple as adding the following to the Apache config:

```
ProxyPass /slskd/ http://the.local.ip.address:5030/slskd/ upgrade=websocket
ProxyPassReverse /slskd/ http://the.local.ip.address:5030/slskd/
```

**Reply by jpdillingham** [reply]
Thanks for following up! I'll add this to the documentation.

**Reply by jpdillingham** [reply]
Closing the loop; the docs for Apache (as well as IIS) were added in #903
